unearthed-arcana/2016/20160912.xml

```diff
@@ -0,0 +1,759 @@
+            <div element="ID_WOTC_UA_CLASS_FEATURE_The Ranger, Revised_PRIMEVAL_AWARENESS" />
+            <div element="ID_WOTC_UA_CLASS_FEATURE_The Ranger, Revised_RANGER_CONCLAVE" />
+            <div element="ID_WOTC_UA_CLASS_FEATURE_ABILITYSCOREIMPROVEMENT_The Ranger, Revised" />
+            <div element="ID_WOTC_UA_CLASS_FEATURE_The Ranger, Revised_RANGER_CONCLAVE_FEATURE" />
+            <div element="ID_WOTC_UA_CLASS_FEATURE_The Ranger, Revised_GREATER_FAVORED_ENEMY" />
+            <div element="ID_WOTC_UA_CLASS_FEATURE_The Ranger, Revised_FLEET_OF_FOOT" />
+           <div element="ID_WOTC_UA_CLASS_FEATURE_The Ranger, Revised_HIDE_IN_PLAIN_SIGHT" />
+           <div element="ID_WOTC_UA_CLASS_FEATURE_The Ranger, Revised_VANISH" />
+           <div element="ID_WOTC_UA_CLASS_FEATURE_The Ranger, Revised_FERAL_SENSES" />
+           <div element="ID_WOTC_UA_CLASS_FEATURE_The Ranger, Revised_FOE_SLAYER" />
+       </description>
+        <sheet display="false">
+      <description>A warrior who uses martial prowess and nature magic to combat threats on the edges of civilization.</description>
+    </sheet>
+    <setters>
+      <set name="short">A warrior who uses martial prowess and nature magic to combat threats on the edges of civilization.</set>
+      <set name="hd">d10</set>
+      <set name="spellcaster">true</set>
+      <set name="spellcastingAbility">Wisdom</set>
+            <set name="sourceUrl">http://dnd.wizards.com/articles/unearthed-arcana/The Ranger, Revised</set>
+        </setters>
+        <sheet display="false"/>
+        <rules>
+        <!-- <grant type="Grants" id="ID_INTERNAL_GRANT_MULTICLASS_SPELLCASTING_SLOTS_HALF" requirements="ID_INTERNAL_GRANT_MULTICLASS"/> -->
+      <!-- <grant type="Grants" name="ID_INTERNAL_GRANTS_AVERAGE_HP_D10" /> -->
+
+      <grant type="Proficiency" name="ID_PROFICIENCY_ARMOR_PROFICIENCY_LIGHT_ARMOR" requirements="!ID_WOTC_UA_MULTICLASS_THE_RANGER,_REVISED" />
+      <grant type="Proficiency" name="ID_PROFICIENCY_ARMOR_PROFICIENCY_MEDIUM_ARMOR" requirements="!ID_WOTC_UA_MULTICLASS_THE_RANGER,_REVISED" />
+      <grant type="Proficiency" name="ID_PROFICIENCY_ARMOR_PROFICIENCY_SHIELDS" requirements="!ID_WOTC_UA_MULTICLASS_THE_RANGER,_REVISED" />
+      <grant type="Proficiency" name="ID_PROFICIENCY_WEAPON_PROFICIENCY_SIMPLE_WEAPONS" requirements="!ID_WOTC_UA_MULTICLASS_THE_RANGER,_REVISED" />
+      <grant type="Proficiency" name="ID_PROFICIENCY_WEAPON_PROFICIENCY_MARTIAL_WEAPONS" requirements="!ID_WOTC_UA_MULTICLASS_THE_RANGER,_REVISED" />
+      <grant type="Proficiency" name="ID_PROFICIENCY_SAVINGTHROW_STRENGTH" requirements="!ID_WOTC_UA_MULTICLASS_THE_RANGER,_REVISED" />
+      <grant type="Proficiency" name="ID_PROFICIENCY_SAVINGTHROW_DEXTERITY" requirements="!ID_WOTC_UA_MULTICLASS_THE_RANGER,_REVISED" />
+      <select type="Proficiency" name="Skill Proficiency (THE RANGER, REVISED)" supports="Skill,Ranger" number="3" requirements="!ID_WOTC_UA_MULTICLASS_THE_RANGER,_REVISED" />
```

Comment:
Format the name of selects simply as **Skill Proficiency (Ranger)** even though it's uppercase in the UI, this might change at some point.